share/models/ingest.py

```diff
@@ -197,6 +197,18 @@ def latest_by_suid_queryset(self, suid_queryset) -> models.QuerySet:
             .values('latest_rawdatum_id')
         ))
 
+    def latest(self) -> models.QuerySet:
```

Comment:
Note: I think overrides the default `QuerySet.latest` function (as in, get latest object in the QS by either `arg0` or `Model.meta.get_latest_by`). If this was not intentional, consider renaming to, e.g. `latest_by_suid` or similar.